src/library.rs

```diff
@@ -250,6 +250,32 @@ impl Fundamental {
             _ => true,
         }
     }
+
+    pub fn get_initialization(self) -> Option<&'static str> {
+        match self {
+            Fundamental::Boolean
+            | Fundamental::Int8
+            | Fundamental::UInt8
+            | Fundamental::Int16
+            | Fundamental::UInt16
+            | Fundamental::Int32
+            | Fundamental::UInt32
+            | Fundamental::Int64
+            | Fundamental::UInt64
+            | Fundamental::Char
+            | Fundamental::UChar
+            | Fundamental::Short
+            | Fundamental::UShort
+            | Fundamental::Int
+            | Fundamental::UInt
+            | Fundamental::Long
+            | Fundamental::ULong
+            | Fundamental::Size
+            | Fundamental::SSize => Some("0"),
```

Comment:
`let b: bool = 0` is not going to work :)

Reply:
Is there any boolean declared anywhere?

Reply:
15 lines above or so in the same block of code

Reply:
At minimum no booleans in gtk-rs